vsintegration/src/FSharp.Editor/CodeFix/CodeFixHelpers.fs

```diff
@@ -26,9 +34,20 @@ module internal CodeFixHelpers =
                 "context.document.project.id", doc.Project.Id.Id.ToString()
                 "context.document.id", doc.Id.Id.ToString()
                 "context.diagnostics.count", diagnostics.Length
+                "context.bulkChange.scope", scope.ToString()
+                "ellapsedMs", ellapsedMs
             ]
 
-        TelemetryReporter.reportEvent "codefixrecommendation" props
+        TelemetryReporter.reportEvent "codefixactivated" props
+
+    let createFixAllProvider name getChangedDocument =
+        FixAllProvider.Create(fun fixAllCtx doc allDiagnostics ->
+            backgroundTask {
+                let sw = Stopwatch.StartNew()
+                let! doc = getChangedDocument (doc, allDiagnostics, fixAllCtx.CancellationToken)
+                do reportCodeFixTelemetry allDiagnostics doc name (fixAllCtx.Scope) sw.ElapsedMilliseconds
```

Comment:
I think Azure telemetry has first-class support of date-time format so we could technically send the date-time string. Currently not sure what's better though.

Reply:
This is a TimeSpan, not a DateTime though.
Will keep just the milliseconds until we see a need for changing it.